app/server/views/login.html

```diff
@@ -18,23 +18,23 @@
 
             <h1 class="govuk-heading-l">{{ i18n.login.header }}</h1>
 
-            <form action="/login" method="post">
+            <form action="/dummy-login" method="post">
```

Comment:
Can you rename this file to `dummy-login` as well, so it's obvious that's it not real?

Reply:
Done